app/pages/index.html

```diff
@@ -29,5 +29,6 @@
   <script src="/scripts/app.js"></script>
   <script src="/scripts/services/Room.js"></script>
   <script src="/scripts/controllers/RoomsCtrl.js"></script>
+  <script src="/node_modules/angular-ui-bootstrap/src/modal/modal.js"></script>
```

Comment:
I used a local path for this. Should it actually be a remote path? Also, should it be higher in the list of script tags? I know the order of the script tags depends on which tag is dependent on the others.